packages/core/src/sdk/overrides/overrides.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Good question, There are 3 errors in override typing in this file. It's very complicated to try to fix things without the full context of the feature. **ts-nocheck** will disable the checking of the entire file, but thinking about it this is not a good alternative as this will omit other typescript checks.

Reply:
It is quite complicated to enable these rules in an ongoing project because we can find files with problems that require some context to resolve. But most of the files were easy to resolve and I believe that this rule will improve our development and the quality of our project. I'll think of a better alternative for this file.

Reply:
I found an alternative to this issue: `@ts-expect-error`

Some definitions:
**Purpose**: To explicitly indicate that you expect a TypeScript error on the following line, and to have TypeScript ensure that an error is present. If no error is found on that line, TypeScript will raise a warning.

**@ts-expect-error** is useful for scenarios where you are working around a known issue, such as a temporary type mismatch that you plan to address later. It helps maintain awareness that the line is supposed to produce an error.

**@ts-ignore** can sometimes be used inappropriately, suppressing errors that you might actually want to address.

Reply:
I think we can use **@ts-expect-error**  before the lines of codes with error and complement it with some explanation of the motivation. This will only affect the line with error. WDTY?

Reply:
I think we can use this approach  here too: https://github.com/vtex/faststore/pull/2379#discussion_r1701780247

Reply:
Yes, I agree